container_registry/container_analysis/src/sample/poll_discovery_finished.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Samples for the Container Analysis golang libraries: https://cloud.google.com/container-registry/docs/container-analysis
+package sample
+
+// [START containeranalysis_poll_discovery_occurrence_finished]
+
+import (
+	"context"
+	"fmt"
+	wait "k8s.io/apimachinery/pkg/util/wait"
```

Comment:
I don't think we want to depend on anything from `k8s.io` - better to show polling code directly.